code/modules/events/brand_intelligence.dm

```diff
@@ -13,8 +13,15 @@
 									"Advertising is legalized lying! But don't let that put you off our great deals!", \
 									"You don't want to buy anything? Yeah, well I didn't want to buy your mom either.")
 
-/datum/event/brand_intelligence/announce()
-	GLOB.minor_announcement.Announce("Rampant brand intelligence has been detected aboard [station_name()], please stand-by. The origin is believed to be \a [originMachine.name].", "Machine Learning Alert", 'sound/AI/brand_intelligence.ogg')
+/datum/event/brand_intelligence/announce(false_alarm)
+	var/alarm_source = originMachine
+	if(originMachine)
+		alarm_source = originMachine.name
+	else
+		var/obj/machinery/economy/vending/vendor_type = pick(subtypesof(/obj/machinery/economy/vending))
+		// TODO WAIT ON THE OTHER VENDOR THING TO GO IN FIRST
```

Comment:
https://github.com/ParadiseSS13/Paradise/pull/22007 will conflict with this, so if the linked PR goes in first, I'll have to update this.